syscalls.go

```diff
@@ -119,8 +121,12 @@ var haveInnerMaps = internal.NewFeatureTest("inner maps", "5.10", func() error {
 		MaxEntries: 1,
 		MapFlags:   unix.BPF_F_INNER_MAP,
 	})
+
 	if err != nil {
-		return internal.ErrNotSupported
+		if errors.Is(err, unix.EINVAL) {
```

Comment:
Ditto, do you want to keep EINVAL here after all?

Reply:
Fixed